src/stories/CardView.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+  import Card from "$lib/components/Card";
+  import type { Variant } from "$lib/components/Card";
+  import Image from "$lib/components/Image";
+  import Icon from "$lib/components/Icon";
+  import { url as arrow } from "$icons/arrow_forward";
+  import Button from "$lib/components/Button";
+
+  import sampleImage from "../uswds-sample.jpg";
+  import imageBlurhash, {
+    width as imageWidth,
+    height as imageHeight,
+    mean as imageMean,
+  } from "../uswds-sample.jpg?blurhash";
+
+  export let variant: Variant = "primary";
+  export let header = "Header";
+  export let body = "Some body text";
+  export let footer = "CTA Button";
+  export let showImage = true;
+  export let showFooter = true;
+  export let tasks: string[] = [];
+</script>
+
+<section class="usa-section">
+  <div class="grid-container">
+    <ul class="usa-card-group">
+      <Card {variant} class="desktop:grid-col-6">
+        <h2 class="usa-card__heading" slot="header">{header}</h2>
+        <div class="image-slot" slot="image">
+          {#if showImage}
+            <Image
+              src={sampleImage}
+              alt="Primary view placeholder"
+              width={imageWidth}
+              height={imageHeight}
+              mean={imageMean}
+              blurhash={imageBlurhash}
+            />
+          {/if}
+        </div>
```

Comment:
nit: I noticed that the following line doesn't seem to be getting covered via tests: https://github.com/la-ldaf/ldaf-site/blob/b7bd4afbdd2763cdd4e0ff08e3b8d1af27f278ac/src/lib/components/Card/Card.svelte#L11

Maybe we should move the `#if` check one line higher to wrap the `slot="image"` container to make sure that we apply the classes properly? Disregard if this messes up the layout or something!

Reply:
Alrighty, it's not the prettiest way of handling it, but since it's only within Storybook, I figure this is an acceptable way to toggle at least the named image and footer slots in 8b16d30d2b6b667d58d954e9560e0771fde820ec and fa64fd0a961192bf45a82f3ef326d6da3e0d3068, respectively